.github/workflows/deploy_production.yml

```diff
@@ -0,0 +1,35 @@
+name: Deploy to AWS (Production)
```

Comment:
New prod deployment workflow. Triggers when merged to `main` and the Unit Tests workflow completes successfully. Same as dev workflow but with `minerva-prod` being chosen for the `cdk deploy` command rather than `minerva-dev`, along with secrets from the production deploy environment being used